src/ts-transformer/css-prop/visitors/visit-jsx-element-with-css-prop.tsx

```diff
@@ -64,16 +69,37 @@ export const visitJsxElementWithCssProp = (
   }
 
   const className = nextClassName();
+  const suppliedClassNameAttribute = getJsxNodeAttributesValue(node, CLASSNAME_PROP);
+  let classNameInitializer;
+
+  if (suppliedClassNameAttribute && ts.isStringLiteral(suppliedClassNameAttribute)) {
+    classNameInitializer = ts.createStringLiteral(
+      `${className} ${suppliedClassNameAttribute.text}`
+    );
+  } else {
+    classNameInitializer = ts.createStringLiteral(className);
+  }
+
+  // if (suppliedClassNameAttribute && ts.isJsxExpression(suppliedClassNameAttribute)) {
+  //   classNameInitializer = ts.createTemplateExpression(
+  //     ts.createTemplateHead(`${className} `, `${className} `),
+  //     ts.createTemplateSpan(suppliedClassNameAttribute.expression!.head, )
```

Comment:
Need to figure out how to append a string to the begging of a JSXExpression 🤔 
[AST viewer here](https://ts-ast-viewer.com/#code/FAegVGAEACBWDOAPSDlhMAlgWwA4HsAnAF0gG8UlIBfSAM0P20gHIASM3AawHMA6AHYBDbAFNqLANzBgAY3wD4pOvnyQAvJAAUASg0A+VivwAjIYSkyAPABNMAN0iyANkPjwAciNHqyAA3k8TGdRGwBaFzd4MOExSA5jXWo-an0AC1FnZzUAdyJnGysQO3t9YEgKyBkgA)